error_writer.go

```diff
@@ -80,12 +80,12 @@ func (w *ErrorWriter) classifyRequest(request *http.Request) protocolType {
 			return unknownProtocol
 		}
 		return connectStreamProtocol
-	case isPost && strings.HasPrefix(ctype, connectUnaryContentTypePrefix):
+	case isPost && strings.HasPrefix(ctype, connectUnaryContentTypePrefix) && !strings.HasPrefix(ctype, grpcContentTypeDefault):
```

Comment:
Should we retain the codec supported is a connect codec here? Even though it is error agnostic it is odd that a request can be classified as a connect request only on the `application/` prefix.

Reply:
Sorry, I don't understand the question.

> odd that a request can be classified as a connect request only on the application/ prefix

This wasn't the case before. To what exactly are you alluding? Do you mean the fact that we'll conflate _any_ `application/` prefix to mean a Connect unary content type? Or are you referring to the check you added to omit the gRPC content-types?

Reply:
Should we remove the config settings for handling grpc and grpc web?